src/cascadia/CommandlineArgs/pch.h

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// pch.h
+// Header for platform projection include files
+//
+
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN
+#define NOMCX
+#define NOHELP
+#define NOCOMM
+
+// Manually include til after we include Windows.Foundation to give it winrt superpowers
+#define BLOCK_TIL
+#include <LibraryIncludes.h>
+// This is inexplicable, but for whatever reason, cppwinrt conflicts with the
+//      SDK definition of this function, so the only fix is to undef it.
+// from WinBase.h
+// Windows::UI::Xaml::Media::Animation::IStoryboard::GetCurrentTime
+#ifdef GetCurrentTime
+#undef GetCurrentTime
+#endif
+
+#include <wil/cppwinrt.h>
+
+#include <unknwn.h>
+
+#include <hstring.h>
+
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Foundation.Collections.h>
+// You might think "we're not doing any UI in this lib", and you'd be right.
+// However, we need this for Windows.UI.Color
+#include <winrt/windows.ui.h>
+
+// Including TraceLogging essentials for the binary
+#include <TraceLoggingProvider.h>
+#include <winmeta.h>
+TRACELOGGING_DECLARE_PROVIDER(g_hTerminalAppProvider);
+#include <telemetry/ProjectTelemetry.h>
+#include <TraceLoggingActivity.h>
+
+#include <shellapi.h>
+#include <shobjidl_core.h>
+
+#include <winrt/Microsoft.Terminal.Settings.Model.h>
+
+#include <CLI11/CLI11.hpp>

```

Comment:
If this is here, why did you have to include it in one of the other files (AppCommandLineArgs.h)?

Reply:
There was a weird compile error. I think one of the other files that consumed the AppCommandlineArgs didn't include CLI11 in their own pch.h, so you'd get compile errors compiling that other project. 

So I think the right solution is just leave it in the header for AppCommandlineArgs, and yank it from this pch.